app/assets/javascripts/controllers/dialog_user/dialog_user_controller.js

```diff
@@ -55,8 +57,20 @@ ManageIQ.angular.app.controller('dialogUserController', ['API', 'dialogFieldRefr
     } else {
       apiData = vm.dialogData;
     }
-    API.post(apiSubmitEndpoint, apiData, {skipErrors: [400]}).then(function() {
-      miqService.redirectBack(__('Order Request was Submitted'), 'info', finishSubmitEndpoint);
+    return API.post(apiSubmitEndpoint, apiData, {skipErrors: [400]})
+      .then(function(response) {
+        if (vm.openUrl === "true") {
+          return API.wait_for_task(response.task_id)
+            .then(function(response) {
+              return $http.post("open_url_after_dialog", {targetId: vm.targetId})
+                .then(function(response) {
+                  window.open(response.data.open_url);
+                  miqService.redirectBack(__('Order Request was Submitted'), 'info', finishSubmitEndpoint);
```

Comment:
Could this be `success` flash message?

![screenshot from 2018-09-24 16-07-55](https://user-images.githubusercontent.com/1187051/45957004-4425b180-c014-11e8-826f-dff41ee601da.png)

Reply:
Done :)